### ISSUE TITLE
minified js ignores version option

### ISSUE THREAD

**Issue by bariscc** [kciter/qart.js#14]
Hi there!
Version option works with qart.js but not with qart.min.js. Seems like it always uses 10.
I receive the following error no matter what version i set:
```
Uncaught Error: code length overflow. (1196>976)
    at C (qart.min.js:1)
    at x (qart.min.js:1)
    at M (qart.min.js:1)
    at Object.A.make (qart.min.js:1)
    at t.value (qart.min.js:1)
    at createQart (remote.js:18)
    at HTMLButtonElement.<anonymous> (remote.js:25)
    at HTMLButtonElement.dispatch (jquery.min.js:3)
    at HTMLButtonElement.q.handle (jquery.min.js:3)
```

Guessing minified js is not up to date.

**Comment by misteral** [comment]
Fixed in #19. Version is up automatically if `code length overflow`

**Comment by misteral** [comment]
@bariscc Hi. Please confirm and feel free to reopen issue if any questions.
